components/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/internal/generator/SingleEventGenerator.java

```diff
@@ -185,21 +191,23 @@ private static SingleEventSimulationDTO createSingleEventDTO(String singleEventC
                             singleEventConfig.toString());
                 }
             }
-            ArrayList dataValues = new Gson().fromJson(singleEventConfig.getJSONArray(EventSimulatorConstants
+            ArrayList dataValues = new ObjectMapper().readValue(singleEventConfig.getJSONArray(EventSimulatorConstants
                     .SINGLE_EVENT_DATA).toString(), ArrayList.class);
-//            create SingleEventSimulationDTO
             SingleEventSimulationDTO singleEventSimulationDTO = new SingleEventSimulationDTO();
             singleEventSimulationDTO.setStreamName(singleEventConfig.getString(EventSimulatorConstants.STREAM_NAME));
             singleEventSimulationDTO.setSiddhiAppName(singleEventConfig
                     .getString(EventSimulatorConstants.EXECUTION_PLAN_NAME));
             singleEventSimulationDTO.setTimestamp(timestamp);
             singleEventSimulationDTO.setAttributeValues(dataValues.toArray());
             return singleEventSimulationDTO;
-        } catch (JSONException e) {
+        }catch (JsonProcessingException e) {
```

Comment:
Can you fix the formatting issue?

Reply:
Fixed with commit [#4eb71832d44025049ade5f9e651eadea3fc556d5](https://github.com/wso2/carbon-analytics/pull/1259/commits/4eb71832d44025049ade5f9e651eadea3fc556d5)